src/middleware/assets.js

```diff
@@ -103,12 +103,17 @@ Or \`rm -rf bower_components/n-ui && bower install n-ui\` if you're no longer wo
 			const nUiUrlRoot = res.locals.flags.nUiBundleMajorVersion ? nUiMajorVersionUrlRoot : nUiSpecificVersionUrlRoot;
 
 			// work out which assets will be required by the page
-			res.locals.nUiConfig = nUiConfig;
-			res.locals.javascriptBundles.push(`\
-${nUiUrlRoot}\
-es5-${res.locals.flags.polyfillSymbol ? 'polyfill-io' : 'core-js'}\
-${(res.locals.flags.nUiBundleUnminified || nUiIsLinked ) ? '' : '.min'}.js`);
-			res.locals.javascriptBundles.push(hashedAssets.get('main-without-n-ui.js'));
+			if (options.hasNUiBundle) {
+				res.locals.nUiConfig = nUiConfig;
+				res.locals.javascriptBundles.push(`\
+				${nUiUrlRoot}\
```

Comment:
having no indentation here is important or it generates a url with lots of whitespace in it. If that's confusing then it might be better to construct the string from an array joined with `''`?